.github/workflows/test.yml

```diff
@@ -0,0 +1,65 @@
+name: Font Awesome SVG CI
+
+on:
+  push:
+    branches:
+      - main
+      - 'stable/**'
+
+  pull_request:
+
+jobs:
+  test-sqlite:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.7', '3.8', '3.9']
+        django: ['3.0', '3.1', '3.2']
+        wagtail: ['2.14', '2.15']
```

Comment:
note to self: we'll need to update this to be 2.15, 4.0 and 4.1. And Django targets accordingly

Reply:
will submit a separate PR once https://github.com/wagtail/cookiecutter-wagtail-package/pull/28, https://github.com/wagtail/cookiecutter-wagtail-package/pull/29 and https://github.com/wagtail/cookiecutter-wagtail-package/pull/32 are in